README.md

```diff
@@ -30,6 +30,20 @@ Flags:
 Use "dockerfmt [command] --help" for more information about a command.
 ```
 
+## Pre-commit
+
+You can add the following entry to your `.pre-commit-config.yaml` file to use
+`dockerfmt` as a pre-commit hook:
+
+```yaml
+repos:
+  - repo: https://github.com/reteps/dockerfmt
+    # run `pre-commit autoupdate` to pin the version
+    rev: main
+    hooks:
+      - id: dockerfmt
```

Comment:
Can you provide an example of how they would change the indent / pass arguments on the README entry?